src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -33,48 +34,51 @@
 import htsjdk.samtools.util.Log;
 import htsjdk.samtools.util.SequenceUtil;
 import htsjdk.variant.utils.SAMSequenceDictionaryExtractor;
+import htsjdk.variant.vcf.VCFFileReader;
+import htsjdk.variant.vcf.VCFHeader;
 import picard.PicardException;
 import picard.analysis.FingerprintingDetailMetrics;
 import picard.analysis.FingerprintingSummaryMetrics;
 import picard.cmdline.CommandLineProgram;
 import picard.cmdline.CommandLineProgramProperties;
 import picard.cmdline.Option;
 import picard.cmdline.StandardOptionDefinitions;
-import picard.cmdline.programgroups.Alpha;
+import picard.cmdline.programgroups.Fingerprinting;
 
 import java.io.File;
-import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
 /**
- * Attempts to check the sample identity of the sequence data in the provided SAM/BAM file
+ * Attempts to check the sample identity of the sequence/genotype data in the provided file (SAM/BAM or VCF)
  * against a set of known genotypes in the supplied genotype file (in either GELI or VCF format).
  *
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
         usage = CheckFingerprint.USAGE_DETAILS,
-        usageShort = "Computes a fingerprint from the supplied SAM/BAM file and compares it to the provided genotypes",
-        programGroup = Alpha.class // TODO -- when mature please move to a to-be-created Fingerprinting.class
+        usageShort = "Computes a fingerprint from the supplied input (SAM/BAM or VCF) file and compares it to the provided genotypes",
+        programGroup = Fingerprinting.class
 )
 public class CheckFingerprint extends CommandLineProgram {
 
-    static final String USAGE_DETAILS = "Computes a fingerprint from the supplied SAM/BAM file and " +
+    static final String USAGE_DETAILS = "Computes a fingerprint from the supplied input file (SAM/BAM or VCF) file and " +
             "compares it to the expected fingerprint genotypes provided. The key output is a LOD score " +
             "which represents the relative likelihood of the sequence data originating from the same " +
             "sample as the genotypes vs. from a random sample.  Two outputs are produced: (1) a summary " +
-            "metrics file that gives metrics related single read group (lane or index within a lane) " +
+            "metrics file that gives metrics at the single sample level (if the input was a VCF) or at the read " +
```

Comment:
vcf's can have more than one sample. please inform here what happens when N!=1 (both 0 and >1)